src/AnalyzeCli.cpp

```diff
@@ -80,8 +95,19 @@ static bool analyzeFile(Luau::Frontend& frontend, const char* name, ReportFormat
         return false;
     }
 
-    for (auto& error : cr.errors)
+    WorkspaceFileResolver* fileResolver = static_cast<WorkspaceFileResolver*>(frontend.fileResolver);
```

Comment:
I don't think we need to static_cast here

Reply:
Actually nevermind, we probably do, since `frontend.fileResolver` doesn't know its a WorkspaceFileResolver